docs/apps/nodes/add-validator.md

```diff
@@ -310,7 +310,7 @@ _**You should only create a keystore user on a node that you operate, as the nod
 
 ### Import Wallet & Node Private Keys
 
-In order to prove that we own the wallet and the node, we need to add our prive keys into the node's keystore.
+In order to prove that we own the wallet and the node, we need to add our private keys into the node's keystore.
```

Comment:
In the sentence oboce we are using "you", and here "we", I suggest that the complete documentation is "you"

Reply:
Fixed